### PR TITLE
deprecate IAM policy page in support for AWS docs

### PR DESCRIPTION
**Pull request by ykethan**
_Issue #, if available:_ https://github.com/aws-amplify/docs/issues/4232, https://github.com/aws-amplify/docs/issues/4702

_Description of changes:_
Deprecate the page in support for AWS IAM documentation page: 
https://docs.aws.amazon.com/amplify/latest/userguide/security-iam-awsmanpol.html#security-iam-awsmanpol-AdministratorAccess-Amplify

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/pages/cli/reference/iam.mdx [hunk 1]
```diff
@@ -245,13 +333,105 @@ The Amplify CLI requires the below IAM policies for performing actions across al
...
+      "Action": ["logs:CreateLogStream", "logs:PutLogEvents"],
+      "Resource": "arn:aws:logs:*:*:log-group:/aws/amplify/*:log-stream:*"
     }
   ]
 }
+
+
 ```
+
+For additional information on AWS managed policies for AWS Amplify refer to [AWS Policy](https://docs.aws.amazon.com/amplify/latest/userguide/security-iam-awsmanpol.html#security-iam-awsmanpol-AdministratorAccess-Amplify)
```
Comment (josefaidt) [inline src/pages/cli/reference/iam.mdx:437]:
```suggestion
For additional information on AWS managed policies for AWS Amplify refer to the [`AdministratorAccess-Amplify` policy](https://docs.aws.amazon.com/amplify/latest/userguide/security-iam-awsmanpol.html#security-iam-awsmanpol-AdministratorAccess-Amplify)
```

File: src/pages/cli/reference/iam.mdx [hunk 2]
```diff
@@ -3,255 +3,4 @@ export const meta = {
...
-        "s3:PutEncryptionConfiguration",
-        "s3:PutObject",
-        "s3:PutObjectAcl"
-      ],
-      "Resource": "*"
-    }
-  ]
-}
-```
+This page has been removed in favor of AWS managed policies for AWS Amplify, refer to the [`AdministratorAccess-Amplify`policy](https://docs.aws.amazon.com/amplify/latest/userguide/security-iam-awsmanpol.html#security-iam-awsmanpol-AdministratorAccess-Amplify)
```
Comment (josefaidt) [inline src/pages/cli/reference/iam.mdx:6]:
```suggestion
This page has been removed in favor of AWS managed policies for AWS Amplify, please refer to the [`AdministratorAccess-Amplify` policy](https://docs.aws.amazon.com/amplify/latest/userguide/security-iam-awsmanpol.html#security-iam-awsmanpol-AdministratorAccess-Amplify)
```

Comment (ykethan) [reply src/pages/cli/reference/iam.mdx]: fixed the wording

**Review by josefaidt — APPROVED** [review]

File: src/pages/cli/reference/iam.mdx [hunk 3]
```diff
@@ -3,255 +3,4 @@ export const meta = {
...
-        "s3:PutEncryptionConfiguration",
-        "s3:PutObject",
-        "s3:PutObjectAcl"
-      ],
-      "Resource": "*"
-    }
-  ]
-}
-```
+This page has been removed in favor of AWS managed policies for AWS Amplify, please refer to the [`AdministratorAccess-Amplify` policy](https://docs.aws.amazon.com/amplify/latest/userguide/security-iam-awsmanpol.html#security-iam-awsmanpol-AdministratorAccess-Amplify)
```
Comment (jhockett) [inline src/pages/cli/reference/iam.mdx:6]:
```suggestion
This page has been removed in favor of AWS managed policies for AWS Amplify. Please refer to the [`AdministratorAccess-Amplify` policy](https://docs.aws.amazon.com/amplify/latest/userguide/security-iam-awsmanpol.html#security-iam-awsmanpol-AdministratorAccess-Amplify).
```

**Review by jhockett — APPROVED** [review]

**Review by jhockett — APPROVED** [review]

**Review by josefaidt — APPROVED** [review]

**Pull request merged**